packages/components/src/Icon/index.tsx

```diff
@@ -1,15 +1,13 @@
 import React from 'react';
 import { icons, IconNames } from '@jpmorganchase/mosaic-theme';
 
-import styles from './styles.css';
-
 export interface IconProps {
   /** Additional class name for root class override. */
   className?: string;
   /** Name of the icon */
   name: IconNames;
   /** Size of Icon */
-  size?: 'small' | 'medium' | 'large';
```

Comment:
Do teams use Icon directly? If so will this cause adoption issues? It it worth supporting both and then mapping to the new values?

Reply:
I had a look in developer docs and couldn't find any uses of size

Reply:
I couldn't find any uses of it in the main docs repo

Reply:
It would be safer to keep this but I don't know if its necessary

Reply:
I think we should keep it for now.